mapbox_vector_tile/__init__.py

```diff
@@ -1,25 +1,24 @@
 from mapbox_vector_tile import decoder, encoder
 
 
-def decode(tile, y_coord_down=False):
-    vector_tile = decoder.TileData()
-    message = vector_tile.get_message(tile, y_coord_down)
+def decode(tile, options=None, default_options=None):
+    vector_tile = decoder.TileData(pbf_data=tile, options=options, default_options=default_options)
+    message = vector_tile.get_message()
     return message
 
 
-def encode(
-    layers,
-    quantize_bounds=None,
-    y_coord_down=False,
-    extents=4096,
-    on_invalid_geometry=None,
-    check_winding_order=True,
-):
-    vector_tile = encoder.VectorTile(extents, on_invalid_geometry, check_winding_order=check_winding_order)
+def encode(layers, options=None, default_options=None):
+    vector_tile = encoder.VectorTile(default_options=default_options)
```

Comment:
nit: can we be more explicit and call it `layer_options` and `default_options`? Or `per_layer_options`?

Also, since we've hoisted the options all under a top level key, what do you think of adding detailed docstrings for the encode/decode functions, and enumerating all options and explaining how they work? I expect that having all the specifics listed out here would go a long way towards making it easy for others to use. Happy to just leave this off for a separate pr though.

Reply:
Could you please read the documentation I wrote? I am not a native English speaker, so you may find some mistakes.

Reply:
this looks great to me, thanks!